triggers/run-completion-event-trigger/server/suite_unit_test.go

```diff
@@ -0,0 +1,15 @@
+//go:build unit
+
+package run_completion_event_trigger
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestRunCompletionEventTrigger(t *testing.T) {
```

Comment:
I think you can move this into `server_unit_test`

Reply:
should probably have a unit test for publishHandler for coverage even though its simple, will add